src/test/fuzz/CMakeLists.txt

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) 2023-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit/.
+
+add_subdirectory(util)
+
+add_executable(fuzz
+  addition_overflow.cpp
+  addrman.cpp
+  asmap.cpp
+  asmap_direct.cpp
+  autofile.cpp
+  banman.cpp
+  base_encode_decode.cpp
+  bech32.cpp
+  bip324.cpp
+  bitdeque.cpp
+  bitset.cpp
+  block.cpp
+  block_header.cpp
+  block_index.cpp
+  blockfilter.cpp
+  bloom_filter.cpp
+  buffered_file.cpp
+  chain.cpp
+  checkqueue.cpp
+  cluster_linearize.cpp
+  coins_view.cpp
+  coinscache_sim.cpp
+  connman.cpp
+  crypto.cpp
+  crypto_aes256.cpp
+  crypto_aes256cbc.cpp
+  crypto_chacha20.cpp
+  crypto_chacha20poly1305.cpp
+  crypto_common.cpp
+  crypto_diff_fuzz_chacha20.cpp
+  crypto_hkdf_hmac_sha256_l32.cpp
+  crypto_poly1305.cpp
+  cuckoocache.cpp
+  decode_tx.cpp
+  descriptor_parse.cpp
+  deserialize.cpp
+  eval_script.cpp
+  feefrac.cpp
+  fee_rate.cpp
+  feeratediagram.cpp
+  fees.cpp
+  flatfile.cpp
+  float.cpp
+  golomb_rice.cpp
+  headerssync.cpp
+  hex.cpp
+  http_request.cpp
```

Comment:
`i2p.cpp` is missing here

Reply:
Thanks!

I don't consider this a blocker, so it will be fixed in a follow-up.

Reply:
Is there a list of known things to be addressed in a followup? It would help us not to forget things and also should avoid people reporting duplicates.

Reply:
we're adding some follow-ups as PRs to https://github.com/hebasto/bitcoin/pulls

Reply:
> we're adding some follow-ups as PRs to https://github.com/hebasto/bitcoin/pulls

Right. This, and https://github.com/bitcoin/bitcoin/issues?q=label%3A%22Needs+CMake+port%22+is%3Aclosed.

Reply:
Comparing the output of `PRINT_ALL_FUZZ_TARGETS_AND_ABORT=1 src/test/fuzz/fuzz` for both autotools and cmake can be used to verify that all harnesses are ported.